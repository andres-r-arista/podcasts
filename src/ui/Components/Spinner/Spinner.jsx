import React from "react";

export const Spinner = () => {
  return (
    <div className="absolute top-2 right-2" data-testid="spinner">
      <img className="h-6 w-6" src="/spinner.gif" alt="Loading..." />
    </div>
  );
};
