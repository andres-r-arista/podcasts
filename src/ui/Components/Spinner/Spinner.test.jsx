import { render } from "@testing-library/react";

import { Spinner } from "./Spinner";

describe("Tests in <Spinner />", () => {
  
  test("must match the snapshot", () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });

});
