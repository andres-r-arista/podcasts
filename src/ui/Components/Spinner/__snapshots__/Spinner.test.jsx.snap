// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <Spinner /> must match the snapshot 1`] = `
<div>
  <div
    class="absolute top-2 right-2"
    data-testid="spinner"
  >
    <img
      alt="Loading..."
      class="h-6 w-6"
      src="/spinner.gif"
    />
  </div>
</div>
`;
